### PR TITLE
fix error function descriptions for alternative negative binomial

### PR DESCRIPTION
**Pull request by mikegilchrist**
- Insert missing 'log' term in `unbounded_discrete_distributions.Rmd` 
- Insert missing ','  in `all.bib`
- Update README.md files with instructions for building individual manuals.

#### Submission Checklist

- [ x] Builds locally
- [ x] New functions marked with `` `r since("VERSION")` ``
- [ x] Declare copyright holder and open-source license: see below

#### Summary

Insert missing 'log' in alternative negative binomial function descriptions 
#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):
 Michael Gilchrist

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC BY-ND 4.0 (https://creativecommons.org/licenses/by-nd/4.0/)

### CODE REVIEW

File: src/stan-users-guide/README.md [hunk 1]
```diff
@@ -1,111 +1,19 @@
-# The Stan Book
+Steps to build functions manual:
```
Comment (WardBrian) [inline src/stan-users-guide/README.md:1]: This line should say “Users guide” instead of function reference

File: src/stan-users-guide/README.md [hunk 2]
```diff
@@ -1,111 +1,19 @@
...
 
-### Directory Structure
+2.  Install on OS and ensure on PATH: pandoc, pandoc-citeproc,
+pdflatex
 
-* `*.Rmd` files: basic text
-* `_bookdown.yml`: book includes
-* `_output.yml`: output config
-* `bib/all.bib`: BibTeX file for references
+3.  To build single manual, such as cmdstan-guide, functions-reference, reference-manual, or stan-users-guide, using Rscript
```
Comment (WardBrian) [inline src/stan-users-guide/README.md:8]: We should really be pointing people toward the top-level build.py script. I’m not sure if the individual READMEs are still that useful at this point

**Review by WardBrian — CHANGES_REQUESTED** [review]
Thanks for the submission! The content and bibtex changes seem good, but I’m not sure about the various README changes. We should probably either delete them entirely or just say to use the build script from the top-level folder

File: src/stan-users-guide/README.md (hunk 1 above)
Comment (mitzimorris) [inline src/stan-users-guide/README.md:1]: This is the Stan users guide, not "functions manual".

File: src/stan-users-guide/README.md [hunk 3]
```diff
@@ -1,111 +1,19 @@
...
+    a. In shell change directory to `docs/src/<MANUAL>`
+    b. In shell, execute 
+       ```
+       # build html
+       > Rscript -e "bookdown::render_book('index.Rmd', output_format='bookdown::gitbook')"
+       # build pdf
+       > Rscript -e "bookdown::render_book('index.Rmd', output_format='bookdown::pdf_book')"
+       ```
+       The output will be written to subdirectory `_build`.
```
Comment (mitzimorris) [inline src/stan-users-guide/README.md:18]: why are you removing this information?

**Comment by mikegilchrist** [comment]
I have deleted the extraneous README.md files.

File: src/stan-users-guide/README.md (hunk 3 above)
Comment (mikegilchrist) [reply src/stan-users-guide/README.md]: Because it is outdated and confusing to a user like me.

**Review by WardBrian — APPROVED** [review]
Thanks!

**Pull request merged**